core-js/src/main/javascript/lib/dataTables/js/jquery.dataTables.js

```diff
@@ -589,11 +589,13 @@
 			else if ( nodeName == 'TD' || nodeName == 'TH' ) {
 				var cell = api.cell( node ).index();
 
-				return [
-					cell.row,
-					cell.columnVisible,
-					cell.column
-				];
+				if (cell !== null && cell !== undefined) {
+					return [
+						cell.row,
+						cell.columnVisible,
+						cell.column
+					];
+				}
```

Comment:
Customizing third party plugins is a bad idea, is updating to a newer version an option?
Or is there an alternative way to tackle this issue?

Reply:
@afrjorge I have checked the latest version. It has the same issue.